extensions-site/app/components/server-card.tsx

```diff
@@ -8,6 +8,16 @@ import { useState } from "react";
 import { motion, AnimatePresence } from "motion/react";
 
 function getGooseInstallLink(server: MCPServer): string {
+  if (server.is_builtin) {
```

Comment:
I think if we show cards with install buttons for the app, it might duplicate the built in extensions in the user's list of installed extensions. Need to think of a solutio for this if we want to include them in the site!

We can consider/discuss tomorrow with a group